tsl/src/bgw_policy/job.c

```diff
@@ -253,6 +276,8 @@ policy_retention_execute(int32 job_id, Jsonb *config)
 
 	policy_retention_read_and_validate_config(config, &policy_data);
 
+	log_retention_window(LOG, &policy_data, "applying retention policy to hypertable");
```

Comment:
this needs to be conditional and only happen when the job is set to verbose

Reply:
you wanna do something similiar like we have for compression job:
tsl/src/bgw_policy/compression_api.c
look for CONFIG_KEY_VERBOSE_LOG

Reply:
@svenklemm interesting, seems the function [`policy_compression_get_verbose_log`](https://github.com/timescale/timescaledb/blob/master/tsl/src/bgw_policy/compression_api.c#L69) is dead code because is not used anywhere

Reply:
Seems we left it behind when @gayyappan change the policy_compression_execute to be a plpgsql procedure in this commit fffd6c2350f5b3237486f3d49d7167105e72a55b